var/spack/repos/builtin/packages/harfbuzz/package.py

```diff
@@ -9,9 +9,10 @@
 class Harfbuzz(AutotoolsPackage):
     """The Harfbuzz package contains an OpenType text shaping engine."""
     homepage = "https://github.com/harfbuzz/harfbuzz"
-    url      = "https://www.freedesktop.org/software/harfbuzz/release/harfbuzz-0.9.37.tar.bz2"
+    url      = "https://github.com/harfbuzz/harfbuzz/releases/download/2.9.1/harfbuzz-2.9.1.tar.xz"
```

Comment:
Are you able to `spack stage harfbuzz@2.6.8`? I'm curious if the updated URL stores the earlier versions with the same hashes.

Reply:
Yes, but that used the cached archive from the spack mirror. I couldn't actually find version 2.6.8 under the old url, but I checked for 2.3.1 and there the checksum for the archive from the old url and from the github release page are identical.

I checked this for the .bz2 archive though - at one point they seemed to have switched from .tar.bz2 to .tar.xz. As long as spack goes  for the bz2 for the old versions we should be good.

Reply:
Oh, I overlooked the `url_for_version` below, so the url change is actually cosmetic here...